docs/guide/static-deploy.md

```diff
@@ -47,13 +47,17 @@ You may configure the port of the server py passing `--port` flag as an argument
 ```json
 {
   "scripts": {
-    "preview": "vite preview --port 8080"
+    "serve": "vite preview --port 8080"
   }
 }
 ```
 
 Now the `preview` method will launch the server at http://localhost:8080.
 
+::: tip NOTE
+If you are using `npm` as your package manager, you may run into issues with changing the `serve` command to `preview` due to npm's usage of `pre` and `post` commands. For more info checkout the [npm docs](https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts) regarding Pre & Post scripts.
```

Comment:
I think we should use script instead of command here. And maybe we shouldn't only pin this to npm (did we test it in yarn 3?). Maybe something like this? 
```suggestion
If you change the script name from `serve` to `preview`, you may run into issues with some package managers due to the way they handle [Pre & Post scripts](https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts).
```